jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt

```diff
@@ -512,31 +512,27 @@ class CodeWhispererService {
         // 2. supplemental context
         val startFetchingTimestamp = System.currentTimeMillis()
         val isTstFile = FileContextProvider.getInstance(project).isTestFile(psiFile)
-        val supplementalContext = if (CodeWhispererUserGroupSettings.getInstance().getUserGroup() == CodeWhispererUserGroup.CrossFile && !isTstFile) {
```

Comment:
remove this `if` block

Reply:
delegate this part of logic to here https://github.com/aws/aws-toolkit-jetbrains/pull/3755/files#diff-7aced45233ad4c7e904e77408bc9f4e8545b011324667d8918129994f555b7a8R114